resources/helm/v1.1/istio_cni/templates/daemonset.yaml

```diff
@@ -0,0 +1,155 @@
+      serviceAccountName: istio-cni
+      # Minimize downtime during a rolling upgrade or deletion; tell Kubernetes to do a "force
+      # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
+      terminationGracePeriodSeconds: 0
+{{- if .Values.priorityClassName }}
+      priorityClassName: "{{ .Values.priorityClassName }}"
+{{- end }}
+{{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+{{- range .Values.imagePullSecrets }}
+      - name: {{ . }}
+{{- end }}
+{{- end }}
+      containers:
+        # This container installs the Istio CNI binaries
+        # and CNI network config file on each node.
+        - name: install-cni-v1_0
+          image: "{{ .Values.image_v1_0 }}"
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          command: ["/install-cni.sh"]
+          env:
+            # Directory where the CNI config file should be created in
+            - name: CNI_NET_DIR
+              value: "/etc/cni/multus/net.d"
+            # Directory where the CNI config file should be created in (host path mounted in the container)
+            - name: MOUNTED_CNI_NET_DIR
+              value: "/host/etc/cni/multus/net.d"
+            # Name of the CNI config file to create.
+            - name: CNI_CONF_NAME
+              value: "istio-cni.conf"
+            # Name of the kubeconfig file used by CNI agent
+            - name: KUBECFG_FILE_NAME
+              value: "istio-cni.kubeconfig"
+            - name: CNI_BINARIES_PREFIX
+              value: ""
+            # Deploy as a standalone CNI plugin instead of as chained
+            - name: CHAINED_CNI_PLUGIN
+              value: "false"
+            # The CNI network config to install on each node.
+            - name: CNI_NETWORK_CONFIG
+              valueFrom:
+                configMapKeyRef:
+                  name: istio-cni-config
+                  key: cni_network_config_1_0
+          volumeMounts:
+            - mountPath: /host/opt/cni/bin
+              name: cni-bin-dir
+            - mountPath: /host/etc/cni/
+              name: etc-cni-dir
+          resources:
+{{- if .Values.resources }}
+{{ toYaml .Values.resources | indent 12 }}
+{{- else }}
+            requests:
+              cpu: 10m
+              memory: 100Mi
+{{- end }}
+        - name: install-cni-v1_1
+          image: "{{ .Values.image_v1_1 }}"
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          command: ["/install-cni.sh"]
+          env:
+            # Directory where the CNI config file should be created in
+            - name: CNI_NET_DIR
+              value: "/etc/cni/multus/net.d"
+            # Directory where the CNI config file should be created in (host path mounted in the container)
+            - name: MOUNTED_CNI_NET_DIR
+              value: "/host/etc/cni/multus/net.d"
+            # Name of the CNI config file to create.
+            - name: CNI_CONF_NAME
+              value: "v1.1-istio-cni.conf"
```

Comment:
@luksa, is this correct?  I wasn't sure if it needed to match the prefixed binary name or not.

Reply:
The name of the NetworkAttachmentDefinition object needs to match the name of the CNI config file. IIRC, the binary name can be configured in the conf file, but defaults to the name of the config file/NAD object name.

Reply:
We're using `v1.1-istio-cni` for the NAD name, so it sounds like we're okay.

https://github.com/Maistra/istio-operator/pull/273/files#diff-a1c55792d1e2085253441f0b5a6d895aR27